apps/server/src/adapters/WebsocketAdapter.ts

```diff
@@ -36,6 +36,10 @@ export class SocketServer implements IAdapter {
   private lastConnection: Date | null = null;
   private shouldShowWelcome = true;
 
+  //should we be tacking versions
```

Comment:
the clients should really have their own version as they get the UI from the server